### PR TITLE
Fix remaining scrapers

### PR DESCRIPTION
**Pull request by flooie**
Fix most of the remaining out scrapers

1. Virgin Islands 
2. Puerto Rico
3. Guam
4. Alabama Supreme
5. Alabama Civ
6. Alabama Crim
7. CA10
8. OLC/AG
9. CGCCA
10. NMCCA
11. ACCA_memorandum
12. ACCA_summary
13. ACCA_Published
14. ASBCA
15. BIA

### CODE REVIEW

**Comment by mlissner** [comment]
Is this all of the busted ones?

**Comment by flooie** [comment]
@mlissner all but `bva`, `miss` and `missappct` and `nyag` I think.  I have a solution for miss and misact and clearly now `minn` is busted with captchas.

File: juriscraper/opinions/united_states/territories/guam.py [hunk 1]
```diff
@@ -58,6 +58,7 @@ def _process_html(self):
                     "url": table.xpath(".//a/@href")[0],
                 }
             )
+            print(self.cases[-1])
```
Comment (quevon24) [inline juriscraper/opinions/united_states/territories/guam.py:61]: The print is required?

Comment (quevon24) [inline juriscraper/opinions/united_states/territories/guam.py:61]:
```suggestion
```

**Review by quevon24 — CHANGES_REQUESTED** [review]
it looks good, only a small change, there is a print in guam scraper

**Pull request merged**